site/source/docs/porting/Debugging.rst

```diff
@@ -26,11 +26,11 @@ Debug information
 
 The *emcc* :ref:`-g flag <emcc-g>` can be used to preserve debug information in the compiled output. By default, this option includes Clang / LLVM debug information in a DWARF format in the generated WebAssembly code and preserves white-space, function names and variable names in the generated JavaScript code.
 
-The flag can also be specified with one of five levels: :ref:`-g0 <emcc-g0>`, :ref:`-g1 <emcc-g1>`, :ref:`-g2 <emcc-g2>`, :ref:`-g3 <emcc-g3>` (default level), and :ref:`-g4 <emcc-g4>`. Each level builds on the last to provide progressively more debug information in the compiled output. See :ref:`Compiler debug information flags <debugging-debug-information-g>` for more details.
+The flag can also be specified with an integer level: :ref:`-g0 <emcc-g0>`, :ref:`-g1 <emcc-g1>`, :ref:`-g2 <emcc-g2>`, and :ref:`-g3 <emcc-g3>` (default level when setting ``-g``). Each level builds on the last to provide progressively more debug information in the compiled output. See :ref:`Compiler debug information flags <debugging-debug-information-g>` for more details.
```

Comment:
This reference - `` :ref:`Compiler debug information flags <debugging-debug-information-g>` `` - doesn't seem right btw, it just links back to the section it's already in?

Reply:
It should probably link to https://emscripten.org/docs/tools_reference/emcc.html#emcc-gn since that provides some more details.

Reply:
Thanks, looks like that was a pre-existing bug here. Fixed.

Reply:
Yeah, not saying it was introduced, just something I spotted while looking at the result.